store/sql_store_test.go

```diff
@@ -0,0 +1,110 @@
+package store
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/go-gorp/gorp"
+	"github.com/go-sql-driver/mysql"
+	"github.com/mattermost/mattermost-mattermod/model"
+)
+
+const (
+	defaultMysqlDSN     = "mattermod:mattermod@tcp(localhost:3306)/mattermost_mattermod_test?charset=utf8mb4,utf8\u0026readTimeout=30s\u0026writeTimeout=30s\u0026parseTime=true"
+	defaultMysqlUser    = "mattermod"
+	defaultMysqlRootPWD = "mattermod"
+	defaultMysqlUserPWD = "mattermod"
+	defaultMysqlDB      = "mattermod_test"
+)
+
+type teardownFunc func()
+
+func getTestSQLStore(t *testing.T) (*SQLStore, teardownFunc) {
```

Comment:
Can we make use of [`t.Cleanup`](https://golang.org/pkg/testing/#T.Cleanup)  here instead of returning a `func()`?

Reply:
Definitely! Thanks @hanzei 👍